.changeset/wise-cheetahs-play.md

```diff
@@ -0,0 +1,6 @@
+---
+'@api3/airnode-deployer': patch
+'@api3/airnode-node': patch
+---
+
+AN-418: Apply timeouts for GCP based on stress tester results
```

Comment:
Can you remove the `AN-418` part? This will be a user-facing changelog and our internal Jira ticket number has no meaning for them.

Reply:
Sure thing :)